code/game/objects/machinery/overwatch.dm

```diff
@@ -25,12 +30,37 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 	var/x_offset_s = 0
 	var/y_offset_s = 0
 	var/living_marines_sorting = FALSE
-	var/busy = FALSE //The overwatch computer is busy launching an OB/SB, lock controls
-	var/dead_hidden = FALSE //whether or not we show the dead marines in the squad.
-	var/z_hidden = 0 //which z level is ignored when showing marines.
-	var/datum/squad/current_squad = null //Squad being currently overseen
-	var/obj/selected_target //Selected target for bombarding
+	var/busy = FALSE ///The overwatch computer is busy launching an OB/SB, lock controls
+	var/dead_hidden = FALSE ///whether or not we show the dead marines in the squad.
+	var/z_hidden = 0 ///which z level is ignored when showing marines.
+	var/datum/squad/current_squad = null ///Squad being currently overseen
+	var/obj/selected_target ///Selected target for bombarding
+	var/current_order = NO_ORDER ///Selected order to give to marine
```

Comment:
this can just be a reference of the current order chosen, no need to be a define.